src/lib.c

```diff
@@ -65,7 +75,7 @@ _efi_set_variable_variadic(efi_guid_t guid, const char *name, const uint8_t *dat
 VERSION(_efi_set_variable_mode,efi_set_variable@@LIBEFIVAR_0.24)
 int NONNULL(2, 3) PUBLIC
 _efi_set_variable_mode(efi_guid_t guid, const char *name, const uint8_t *data,
-		       size_t data_size, uint32_t attributes, mode_t mode)
+			   size_t data_size, uint32_t attributes, mode_t mode)
```

Comment:
You shouldnt change tabbing on this patch

Reply:
Done